java/tomcat/debian/obm-tomcat.init

```diff
@@ -36,7 +36,7 @@ if [ -r /etc/default/rcS ]; then
 fi
 
 if [ -z "$JAVA_HOME" ]; then
-	JAVA_HOME="/usr/lib/jvm/java-7-openjdk-"`dpkg --print-architecture`
+	JAVA_HOME="usr"
```

Comment:
java_home can't really be defined as usr because it expect a specific directory layout.
It has to be `/usr/lib/jvm/something`